mne/io/egi/egi.py

```diff
@@ -242,28 +236,23 @@ def __init__(self, input_fname, montage=None, eog=None, misc=None,
                     raise ValueError('`%s` must be None or of type list' % kk)
 
             event_ids = np.arange(len(include_)) + 1
-            try:
-                logger.info('    Synthesizing trigger channel "STI 014" ...')
-                logger.info('    Excluding events {%s} ...' %
-                            ", ".join([k for i, k in enumerate(event_codes)
-                                       if i not in include_]))
-                new_trigger = _combine_triggers(egi_events[include_],
-                                                remapping=event_ids)
-                data = np.concatenate([data, new_trigger])
-            except RuntimeError:
```

Comment:
If you look at the old `_combine_triggers` code there was one place it explicitly raised `RuntimeError` and I have it return `None` now. If there were other places it implicitly did it, then we should refactor it anyway because it's nasty to try to figure out of from the code, but I don't think there were.